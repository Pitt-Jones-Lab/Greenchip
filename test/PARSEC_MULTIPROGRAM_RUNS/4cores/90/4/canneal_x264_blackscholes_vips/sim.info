{'benchmarks_revision': ('cat: .gitid: No such file or directory',),
 'benchmarks_root': ('/homes/grads/dek61/SNIPER/benchmarks',),
 'cmdline': ['/homes/grads/dek61/SNIPER/sniper/run-sniper',
             '-n',
             '4',
             '-m',
             'localhost',
             '-d',
             '/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/90/4/canneal_x264_blackscholes_vips',
             '-c',
             '/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_90_4_4.cfg',
             '-s',
             'stop-by-icount:10000000000',
             '--curdir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/90/4/canneal_x264_blackscholes_vips',
             '--trace-manual',
             '-g',
             '--traceinput/enabled=true',
             '-g',
             '--traceinput/emulate_syscalls=true',
             '-g',
             '--traceinput/num_apps=4',
             '-g',
             '--traceinput/trace_prefix=/tmp/tmpoNMqfE/run_benchmarks'],
 'git_revision': '68418dbe7cab15c65f1f4d300da0c505329beedd',
 'host': 'sb8',
 'pin_version': 'unknown',
 'rusage': (17379.957844,
            846.28534500000001,
            270448,
            0,
            0,
            0,
            64870,
            2,
            0,
            10720,
            1288,
            0,
            0,
            0,
            104098224,
            5774231),
 'snipercmd': ['bash',
               '-c',
               '/homes/grads/dek61/SNIPER/sniper/lib/sniper -c /homes/grads/dek61/SNIPER/sniper/config/base.cfg --general/total_cores=4 --general/output_dir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/90/4/canneal_x264_blackscholes_vips --config=/homes/grads/dek61/SNIPER/sniper/config/nehalem90.cfg --config=/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_90_4_4.cfg --traceinput/enabled=true --traceinput/emulate_syscalls=true --traceinput/num_apps=4 --traceinput/trace_prefix=/tmp/tmpoNMqfE/run_benchmarks -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/90/4/canneal_x264_blackscholes_vips/sim.scripts.py -g --hooks/script0args='],
 't_elapsed': 7977.1175701618195,
 't_start': 1464034800.254988,
 'tracecmds': [],
 'user': 'nbp3',
 'vmem': 1687109632L}
