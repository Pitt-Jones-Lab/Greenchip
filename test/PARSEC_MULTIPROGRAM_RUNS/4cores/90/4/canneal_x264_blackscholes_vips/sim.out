                                   | Core 0      | Core 1      | Core 2      | Core 3     
  Instructions                     |  1701366325 |  2179278936 |  2798993446 |  3319003511
  Cycles                           |  1466556420 |  1466556420 |  1466556420 |  1466556420
  IPC                              |        1.16 |        1.49 |        1.91 |        2.26
  Time (ns)                        |   551337000 |   551337000 |   551337000 |   551337000
  Idle time (ns)                   |       95601 |       64098 |      256570 |     1001095
  Idle time (%)                    |        0.0% |        0.0% |        0.0% |        0.2%
Branch predictor stats             |             |             |             |            
  num correct                      |   166517680 |   124720696 |   329288012 |   304074051
  num incorrect                    |      227520 |     6921424 |     9414147 |     1366517
  misprediction rate               |       0.14% |       5.26% |       2.78% |       0.45%
  mpki                             |        0.13 |        3.18 |        3.36 |        0.41
TLB Summary                        |             |             |             |            
  I-TLB                            |             |             |             |            
    num accesses                   |   221077147 |   249479773 |   382285975 |   409353119
    num misses                     |        1829 |       11575 |        5485 |        7793
    miss rate                      |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                           |        0.00 |        0.01 |        0.00 |        0.00
  D-TLB                            |             |             |             |            
    num accesses                   |   550033039 |   702630099 |   806930658 |   932099510
    num misses                     |      176903 |     2634677 |      288340 |     2307779
    miss rate                      |       0.03% |       0.37% |       0.04% |       0.25%
    mpki                           |        0.10 |        1.21 |        0.10 |        0.70
  L2 TLB                           |             |             |             |            
    num accesses                   |      178732 |     2646252 |      293825 |     2315572
    num misses                     |      123062 |      390051 |       83304 |      294327
    miss rate                      |      68.85% |      14.74% |      28.35% |      12.71%
    mpki                           |        0.07 |        0.18 |        0.03 |        0.09
Cache Summary                      |             |             |             |            
  Cache L1-I                       |             |             |             |            
    num cache accesses             |   221077147 |   249479773 |   382285975 |   409353119
    num cache misses               |       75293 |    12013769 |      194259 |     3734971
    miss rate                      |       0.03% |       4.82% |       0.05% |       0.91%
    mpki                           |        0.04 |        5.51 |        0.07 |        1.13
  Cache L1-D                       |             |             |             |            
    num cache accesses             |   550033039 |   702630099 |   806930658 |   932099510
    num cache misses               |     8427522 |    11745648 |     7233094 |    14501707
    miss rate                      |       1.53% |       1.67% |       0.90% |       1.56%
    mpki                           |        4.95 |        5.39 |        2.58 |        4.37
  Cache L2                         |             |             |             |            
    num cache accesses             |    12372061 |    24570145 |     8534573 |    19111708
    num cache misses               |     5903822 |     4226209 |     2660410 |     2840524
    miss rate                      |      47.72% |      17.20% |      31.17% |      14.86%
    mpki                           |        3.47 |        1.94 |        0.95 |        0.86
DRAM summary                       |             |             |             |            
  num dram accesses                |    28485604 |           0 |           0 |           0
  average dram access latency (ns) |       93.65 |         inf |         inf |         inf
  average dram queueing delay      |       93.65 |        0.00 |        0.00 |        0.00
Coherency Traffic                  |             |             |             |            
  num loads from dram              |     4938404 |     3252417 |     2473409 |     2590637
  num loads from dram cache        |           0 |           0 |           0 |           0
  num loads from remote cache      |           0 |           0 |           0 |           0
